packages/http/src/validator/validators/params.ts

```diff
@@ -29,7 +29,7 @@ export class HttpParamsValidator<Target, Spec extends IHttpParam> implements IHt
       }
 
       const resolvedStyle = spec.style || style;
-      if (spec.content && spec.content.schema && target[spec.name]) {
+      if (spec.content && spec.content.schema && validateJSONSchema(spec.content.schema) && target[spec.name]) {
```

Comment:
why not just: `ajv.validateSchema(spec.content.schema)`
introducing `validateJSONSchema` looks a bit too much for this simple case

Reply:
The problem is that the `ajv` instance in that function is shared and configured ad hoc. For example, it includes the draft-4 meta schema which is required for multiple parts of OAS2 validations. We need to use exactly that instance.

Does that answer your question?

Reply:
Yes, thank you ! :)